docs/technical.rst

```diff
@@ -7,210 +7,144 @@ Technical Reference
 This guide gives an overview on common practices and technical standards
 shared within the ``EthereumJS`` ecosystem.
 
-Source Code
+Development
 ===========
 
-Development Runtime
--------------------
+Node.js
+-------
 
-Node.js Version Support
-^^^^^^^^^^^^^^^^^^^^^^^
+Development Version
+^^^^^^^^^^^^^^^^^^^
 
 Runtime environment for development is `node.js <https://nodejs.org/en/>`_.
 
-Development should always be possible running the last two ``LTS`` Node.js versions,
+Development should always be possible running latest ``LTS`` Node.js version,
 see Node.js `release schedule table <https://github.com/nodejs/Release#release-schedule>`_.
 
 ====================== ================= ===============================
 Node Version           Status            Latest Status Change
 ====================== ================= ===============================
-Node.js 8              Supported         2018-11-01
-Node.js 10             Supported         2018-11-01
+Node.js 12              Supported         2020-10-19
 ====================== ================= ===============================
 
-Node.js Features
-^^^^^^^^^^^^^^^^
+package-lock Files
+^^^^^^^^^^^^^^^^^^
 
-Buffer vs safe-buffer
-"""""""""""""""""""""
-Old Node versions up to versions ``4`` and ``5`` allowed for some unsafe usage
-of the ``Buffer`` API which led to the development of a replacement 
-`safe-buffer <https://github.com/feross/safe-buffer>`_ library. We have now for
-some time dropped support for distribution on all affected Node versions (mainly
-``4``) and usage of ``safe-buffer`` is not needed any more.
+**Standalone Libraries**
 
-.. note::
-   The ``safe-buffer`` is still in 
-   use `on many libraries <https://github.com/search?q=org%3Aethereumjs+safe-buffer&type=Code>`_,
-   this should be removed and updated with direct ``Buffer`` usage.
+Do not use lock files (``package-lock.json``) on repositories 
+(instead add to ``.gitignore``), see 
+`this <https://github.com/ethereumjs/merkle-patricia-tree/pull/62>`_ discussion
```

Comment:
I think @alcuadrado recently shared that he has heard some good arguments including package-locks, which we now do in the monorepo (one improvement is in speed/caching). The hoisting note below for the monorepo is more that the individual packages don't have package-locks since the one in the root covers them all.

I think we can just remove this "package-lock Files" section, or just note that it was previously agreed that package-locks are not strictly necessary for the libraries but is recently under reconsideration due to other benefits (speed, caching, reliability, reproducibility, etc.) and is taken on a case-by-case basis.

One way we detect possible issues from newly updated dependencies is running the vm nightly test with a fresh package-lock.

Reply:
Thanks 😄 , have updated the section.